content/en/docs/instrumentation/php/sdk.md

```diff
@@ -117,7 +117,7 @@ There are also a number of PHP-specific configurations:
 | OTEL_PHP_TRACES_PROCESSOR          | batch         | batch, simple                                                         | simple         | Span processor selection                            |
 | OTEL_PHP_DETECTORS                 | all           | env, host, os, process, process_runtime, sdk, sdk_provided, container | env,os,process | Resource detector selection                         |
 | OTEL_PHP_AUTOLOAD_ENABLED          | false         | true, false                                                           | true           | Enable/disable SDK autoloading                      |
-| OTEL_PHP_DISABLED_INSTRUMENTATIONS | []            | Instrumentation name(s)                                               | psr15,psr18    | Disable one or more installed auto-instrumentations |
+| OTEL_PHP_DISABLED_INSTRUMENTATIONS | []            | Instrumentation name(s)                                               | psr15,PSR-18    | Disable one or more installed auto-instrumentations |
```

Comment:
It's not picking up `psr15`, maybe because of the missing space in the list?

```suggestion
| OTEL_PHP_DISABLED_INSTRUMENTATIONS | []            | Instrumentation name(s)                                               | psr15, PSR-18    | Disable one or more installed auto-instrumentations |
```

Reply:
(Maybe there isn't supposed to be a space?)